### PR TITLE
Implemented Numeric#positive? and Numeric#negative?

### PR DESCRIPTION
**Pull request by Locke23rus**

### CODE REVIEW

**Comment by jc00ke** [comment]
There are a mix of styles in the specs...

```ruby
x.negative?.should == false
# and
x.negative?.should be_false
```

We should use just one style.

File: spec/ruby/core/bignum/negative_spec.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe "Bignum#negative?" do
+  it "returns true if self is less than 0" do
```
Comment (jc00ke) [inline spec/ruby/core/bignum/negative_spec.rb]: The `x.negative?.should == false` expectations don't exercise what the spec description describes. :confused:

**Comment by Locke23rus** [comment]
@jc00ke which style should I use `== true/false`?

File: spec/ruby/core/float/negative_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      +1.0,
+      +0.0,
+      -0.0,
+      +(0.0.next_float),
+      +Float::INFINITY,
+      Float::NAN
+    ]
+
+    not_negative_numbers.each do |number|
+      number.negative?.should == false
```
Comment (jc00ke) [inline spec/ruby/core/float/negative_spec.rb]: This expectation doesn't exercise the description either.

**Comment by jc00ke** [comment]
@Locke23rus I see `be_[true|false]` is used in `Integer` specs (`Integer#even?` for instance) but I see `should == false` used in other places. I'm partial to `be_[true|false]` personally.

**Comment by Locke23rus** [comment]
@jc00ke thanks, request is updated.

**Comment by brixen** [comment]
Thanks!

**Pull request merged**